packages/web/app/modules/nominee-flow/sagas.ts

```diff
@@ -585,7 +585,10 @@ export function* setActiveNomineeEto({ logger }: TGlobalDependencies): Generator
           ),
         );
       } else {
-        const firstEto = nonNullable(Object.values(etos)[0]);
+        const firstEto = nonNullable(
+          Object.values(etos)[0],
+          "`setActiveNomineeEto` requires `firstEto`",
```

Comment:
```suggestion
          "`setActiveNomineeEto` requires `etos[0]`", 
```
since firstEto is not a variable that exists